week2/tests/test_charIsIn.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import os
+import sys
+import solutions.charIsIn as charIsIn
+
```

Comment:
Imports should always be lexically sorted, and in 3 sections:
(1) System libraries + a newline
(2) 3rd party libraries + a newline
(3) Your project's imports + a newline